src/i18n.js

```diff
@@ -41,7 +41,9 @@ const LanguageSelect = () => {
   return (
     <Select
       options={LANGUAGE_OPTIONS}
-      value={LANGUAGE_OPTIONS.find((option) => option.value === i18n.language)}
+      value={LANGUAGE_OPTIONS.find(
```

Comment:
I didn't know i18n.languages is populated, does i18n.language ever have something different from i18n.languages[0]?

Reply:
`i18n.language` is the current language (e.g. `zh-tw`), not the resolved language from the list of supported languages (e.g. `zh-hant`). i18n.languages are the selected supported languages (in fallback order) (e.g. [`zh-hant`, `en`].